src/GovUk.Education.ExploreEducationStatistics.Content.Security/AuthorizationHandlers/ViewReleaseAuthorizationHandler.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Microsoft.AspNetCore.Authorization;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Security.AuthorizationHandlers
+{
+    public class ViewReleaseRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class ViewReleaseAuthorizationHandler
+        : AuthorizationHandler<ViewReleaseRequirement, Release>
+    {
+        public ViewReleaseAuthorizationHandler()
+        {
+        }
+
+        protected override Task HandleRequirementAsync(
+            AuthorizationHandlerContext authContext,
+            ViewReleaseRequirement requirement,
+            Release release)
+        {
+            if (release.Live)
```

Comment:
This will still pass if it's a Release that's been superseded by an amendment that's been published since `Live` is a little misleading and it's no longer live. I recently added extension `release.IsLatestPublishedVersionOfRelease()` which you could use here but it needs the Publication to be hyrdated to use it.

Reply:
Good catch! I've added in another query to ensure that the hydrated release is fetched and then we now run the `IsLatestPublishedVersionOfRelease` method on it.